kubernetes-client/pom.xml

```diff
@@ -259,6 +265,37 @@
       <groupId>org.awaitility</groupId>
       <artifactId>awaitility</artifactId>
     </dependency>
+      <dependency>
+          <groupId>io.fabric8</groupId>
+          <artifactId>kubernetes-model-core</artifactId>
+          <version>5.1.1</version>
+      </dependency>
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-model-apps</artifactId>
+      <version>5.1.1</version>
```

Comment:
I think the version should be used here.

Reply:
why are these changes required?

Reply:
removed `pom.xml` completely.